Core/include/Acts/Seeding/HoughTransformUtils.hpp

```diff
@@ -220,10 +241,23 @@ class HoughPlane {
   /// @brief get the list of cells with non-zero content.
   /// Useful for peak-finders in sparse data
   /// to avoid looping over all cells
-  const std::set<std::pair<std::size_t, std::size_t>>& getNonEmptyBins() const {
+  const std::unordered_set<std::size_t>& getNonEmptyBins() const {
     return m_touchedBins;
   }
 
+  /// @brief get the coordinates of the bin given the global bin index
+  /// @param globalBin: the global bin index of the cell
+  std::array<std::size_t, 2> axisBins(std::size_t globalBin) const {
```

Comment:
suggesting an std::pair instead return type.
Or a ROOT-like void axisBins(size_t global, size_t & x, size_t & y) signature, which would prevent copies / assignment overhead

Reply:
Pair will take as much memory as an array and confuses me more.

Reply:
do you plan to iterate or run std algorithms over the axis bins? Otherwise, why add the bells and whistles of an array when what we represent is a pair of indices?